npc/re/mobs/dungeons/for_dun.txt

```diff
@@ -0,0 +1,117 @@
+for_dun01	monster	--ja--	22205,16	// SPIRIT_I_WATER_M
+for_dun01	monster	--ja--	22206,18	// SPIRIT_I_WATER_L
+for_dun01	monster	--ja--	22195,4	// SPIRIT_G_LAND_SL
+for_dun01	monster	--ja--	22199,4	// SPIRIT_B_FLAME_SL
+for_dun01	monster	--ja--	22203,4	// SPIRIT_S_WIND_SL
+for_dun01	monster	--ja--	22207,4	// SPIRIT_I_WATER_SL
+
+for_dun02	monster	--ja--	22216,24	// SPIRIT_H_WATER_S
+for_dun02	monster	--ja--	22217,28	// SPIRIT_H_WATER_M
+for_dun02	monster	--ja--	22218,32	// SPIRIT_H_WATER_L
+for_dun02	monster	--ja--	22220,24	// SPIRIT_D_WIND_S
+for_dun02	monster	--ja--	22221,28	// SPIRIT_D_WIND_M
+for_dun02	monster	--ja--	22222,32	// SPIRIT_D_WIND_L
+for_dun02	monster	--ja--	22224,24	// SPIRIT_R_FLAME_S
+for_dun02	monster	--ja--	22225,28	// SPIRIT_R_FLAME_M
+for_dun02	monster	--ja--	22226,32	// SPIRIT_R_FLAME_L
+for_dun02	monster	--ja--	22228,24	// SPIRIT_F_LAND_S
+for_dun02	monster	--ja--	22229,28	// SPIRIT_F_LAND_M
+for_dun02	monster	--ja--	22230,32	// SPIRIT_F_LAND_L
+for_dun02	monster	--ja--	22219,7	// SPIRIT_H_WATER_SL
+for_dun02	monster	--ja--	22223,7	// SPIRIT_D_WIND_SL
+for_dun02	monster	--ja--	22227,7	// SPIRIT_R_FLAME_SL
+for_dun02	monster	--ja--	22231,7	// SPIRIT_F_LAND_SL
+
+t_garden,1,1,0	script	control_t_garden	HIDDEN_WARP_NPC,{
+	end;
+OnInit:
+OnMinute59:
+	.zone = !.zone;
+
+	if (.zone == 0) {	// Zone 2 (for_dun01) -> Zone 1 (for_dun02)
+		mapannounce "t_garden", "In approximately 1 minute, the dimensional contamination in Area 2 will subside and a large-scale dimensional contamination will occur in Area 1.", bc_map, 0xCCFF00;
+		mapannounce "for_dun02", "Large-scale dimensional contamination event will occur in this dungeon in approximately 1 minute.", bc_map, 0xFFDD;
+		mapannounce "for_dun01", "The dimensional contamination phenomenon in this dungeon is expected to subside in about 1 minute.", bc_map, 0xFFDD;
+	}
+	else {	// Zone 1 (for_dun02) -> Zone 2 (for_dun01)
+		mapannounce "t_garden", "In approximately 1 minute, the dimensional contamination in Area 1 will subside and a large-scale dimensional contamination will occur in Area 2.", bc_map, 0xCCFF00;
+		mapannounce "for_dun02", "The dimensional contamination phenomenon in this dungeon is expected to subside in about 1 minute.", bc_map, 0xFFDD;
+		mapannounce "for_dun01", "Large-scale dimensional contamination event will occur in this dungeon in approximately 1 minute.", bc_map, 0xFFDD;
+	}
+	initnpctimer;
+	end;
+
+OnTimer60000:
+	stopnpctimer;
+
+	for ( .@i = 22208; .@i <= 22215; ++.@i )
+		killmonster "for_dun01", "control_t_garden::OnMob" + .@i;
+	for ( .@i = 22232; .@i <= 22239; ++.@i )
+		killmonster "for_dun02", "control_t_garden::OnMob" + .@i;
+
+	if (.zone == 0) {	// Zone 2 (for_dun01) -> Zone 1 (for_dun02)
+		mapannounce "t_garden", "Forgotten Memories The dimensional contamination phenomenon in Area 2 has calmed down, and a large-scale dimensional contamination phenomenon has occurred in Area 1.", bc_map, 0xFFDD;
```

Comment:
Is "Forgotten Memories" a name or something specific? Is it supposed to be separated by a ":" or something to show it's a title/etc?